integration-tests/oidc-wiremock/src/test/java/io/quarkus/it/keycloak/CodeFlowAuthorizationTest.java

```diff
@@ -279,16 +279,16 @@ public void testCodeFlowUserInfo() throws Exception {
         wireMockServer.resetRequests();
         // No internal ID token
         doTestCodeFlowUserInfo("code-flow-user-info-only", 300, false, false, 1, 1);
-        clearCache();
+        //clearCache();
         // Internal ID token, allow in memory cache = true, cacheUserInfoInIdtoken = false without having to be configured
-        doTestCodeFlowUserInfo("code-flow-user-info-github", 25200, false, false, 1, 1);
-        clearCache();
+        //doTestCodeFlowUserInfo("code-flow-user-info-github", 25200, false, false, 1, 1);
+        //clearCache();
         // Internal ID token, allow in memory cache = false, cacheUserInfoInIdtoken = true without having to be configured
-        doTestCodeFlowUserInfo("code-flow-user-info-dynamic-github", 301, true, true, 0, 1);
-        clearCache();
+        //doTestCodeFlowUserInfo("code-flow-user-info-dynamic-github", 301, true, true, 0, 1);
+        //clearCache();
         // Internal ID token, allow in memory cache = false, cacheUserInfoInIdtoken = false
-        doTestCodeFlowUserInfo("code-flow-user-info-github-cache-disabled", 25200, false, false, 0, 4);
-        clearCache();
+        //doTestCodeFlowUserInfo("code-flow-user-info-github-cache-disabled", 25200, false, false, 0, 4);
+        //clearCache();
```

Comment:
Why all these commented lines?

Reply:
@gsmet Sorry, I think I was debugging it and forgot to remove comments, thanks for spotting it, I'll need to refresh this PR

Reply:
@gsmet Hi, I've re-enabled this code which I forgot to uncomment...